OERCONE/iocBoot/iocOERCONE-IOC-01/config.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
+<config_part>
+<ioc_desc>The Oerlikon Centre-One Pressure Gauge</ioc_desc>
+<macros>
+<macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM port" />
```

Comment:
According to the manual, the baud rate is variable between 9600/19200/38400, so that should be an option in this list.

Reply:
All changed